### PR TITLE
feat: deleteIdentityPermissions implemented

### PR DESCRIPTION
**Pull request by htztomic**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by DaveMadsen — APPROVED** [review]

File: workbench-core/authorization/README.md [hunk 1]
```diff
@@ -5,7 +5,11 @@
 # Code Coverage
 | Statements                  | Branches                | Functions                 | Lines             |
 | --------------------------- | ----------------------- | ------------------------- | ----------------- |
+<<<<<<< HEAD
```
Comment (DaveMadsen) [inline workbench-core/authorization/README.md:8]: This merge conflict was committed, but not resolved.

**Review by DaveMadsen — CHANGES_REQUESTED** [review]
actually fix the merge conflict.

File: workbench-core/authorization/README.md (hunk 1 above)
Comment (htztomic) [reply workbench-core/authorization/README.md:8]: odd, vscode didn't overwrite the changes

Comment (htztomic) [reply workbench-core/authorization/README.md]: fixed

**Review by DaveMadsen — APPROVED** [review]

**Pull request merged**